manticore/native/cpu/x86.py

```diff
@@ -5881,6 +5881,31 @@ def PSUBB(cpu, dest, src):
             result.append((a - b) & 0xFF)
         dest.write(Operators.CONCAT(8 * len(result), *result))
 
+    @instruction
+    def PSUBQ(cpu, dest, src):
+        """
+        PSUBQ: Packed add with quadruple words
+        Packed subtract with quad
+
+        Subtracts the second operand (source operand) from the first operand (destination operand) and stores
+        the result in the destination operand. When packed quadword operands are used, a SIMD subtract is performed.
+        When a quadword result is too large to be represented in 64 bits (overflow), the result is wrapped around
+        and the low 64 bits are written to the destination element (that is, the carry is ignored).
+
+        :param cpu: current CPU.
+        :param dest: destination operand.
+        :param src: source operand.
+        """
+        result = []
+        value_a = dest.read()
+        value_b = src.read()
+
+        for i in reversed(range(0, dest.size, 64)):
+            a = Operators.EXTRACT(value_a, i, 64)
+            b = Operators.EXTRACT(value_b, i, 64)
+            result.append((a - b) & 0xFFFFFFFFFFFFFFFF)
```

Comment:
We are subtracting on 64 bits already so I don't think masking with `0xffffffffffffffff` is necessary?

Reply:
Ok! I thought it should be used to explicitly mask any negative results, but I think this is also done in `CONCAT` so I will remove it.